### ISSUE TITLE
How to identify packages associated with a class file

### ISSUE THREAD

**Issue by PhoenixMallu** [soot-oss/soot#1916]
Can anyone suggest a technique to identify the package name from class file?

**Comment by jpstotz** [comment]
Have you tried `SootClass#getJavaPackageName()`?

**Comment by PhoenixMallu** [comment]
Yes, I tried that ... its showing package as null... I want to segregate between two classes with same name as part of static analysis...
